etcdutl/etcdutl/backup_command.go

```diff
@@ -167,32 +167,20 @@ func saveSnap(lg *zap.Logger, destSnap, srcSnap string, desired *desiredCluster)
 		walsnap.Index, walsnap.Term, walsnap.ConfState = snapshot.Metadata.Index, snapshot.Metadata.Term, &desired.confState
 		newss := snap.New(lg, destSnap)
 		snapshot.Metadata.ConfState = desired.confState
-		snapshot.Data = mustTranslateV2store(lg, snapshot.Data, desired)
+		snapshot.Data = mustTranslateV2store(lg, desired)
```

Comment:
Why are we changing `etcdutl backup` command at all? It's a legacy deprecated way to create snapshot.

Reply:
This set of PRs is going towards removing the SetStore method of RaftCluster (along with usage of that store). The backup command is a consumer of SetStore so it needs to be updated. 

If the backup command needs to be removed from code base, I can open another issue to get that done and inform the user base that the backup command is going away in 3.6. Please let me know if the backup command should be removed. /cc @ahrtr

Reply:
created:https://github.com/etcd-io/etcd/issues/16321

Reply:
> This set of PRs is going towards removing the SetStore method of RaftCluster (along with usage of that store). The backup command is a consumer of SetStore so it needs to be updated.


Agreed. I need to update the `etcdutl backup` command, otherwise we have to keep some source code (e.g. SetStore) which are used by etcdutl only.

> Please let me know if the backup command should be removed. /cc @ahrtr

Let's discuss it separately. I tend to remove it.